tests/use_cases/repositories.py

```diff
@@ -994,17 +993,33 @@ def get_available_language_codes(self) -> Iterable[str]:
 
 @singleton
 class FakePayoutFactorRepository:
+    @dataclass
+    class _PayoutFactorModel:
+        factor: PayoutFactor
+
+        def __lt__(self, other: FakePayoutFactorRepository._PayoutFactorModel) -> bool:
+            return (
+                self.factor.calculation_date < other.factor.calculation_date
+                and self.factor.value < other.factor.value
```

Comment:
why should the factor value play a role here if we want to have a iterable sorted by calculation time?